pages/services/kafka-zookeeper/2.6.0-3.4.14/security/index.md

```diff
@@ -0,0 +1,299 @@
+        "security": {
+            "kerberos": {
+                "advanced": {
+                    "required_for_quorum_learner": true,
+                    "required_for_quorum_server": false,
+                    "required_for_client": false
+                }
+            }
+        }
+    }
+}
+```
+and deploy this as a configuration update:
+```bash
+$ dcos kafka-zookeeper --name=<service name> update start --options=kerberos-toggle-step-2.json
+$ dcos kafka-zookeeper --name=<service name> plan show deploy
+deploy (serial strategy) (COMPLETE)
+└─ node-update (serial strategy) (COMPLETE)
+   ├─ zookeeper-0:[server, metrics] (COMPLETE)
+   ├─ zookeeper-1:[server, metrics] (COMPLETE)
+   └─ zookeeper-2:[server, metrics] (COMPLETE)
+```
+deploying a {{ model.techName }} instance that *requires* Kerberos authentication between learners in the leader election.
+
+As the next step in the rolling update process, create a `kerberos-toggle-step-3.json` file with the following contents:
+```json
+{
+    "service": {
+        "security": {
+            "kerberos": {
+                "advanced": {
+                    "required_for_quorum_learner": true,
+                    "required_for_quorum_server": true,
+                    "required_for_client": false
+                }
+            }
+        }
+    }
+}
+```
+and deploy this as a configuration update:
+```bash
+$ dcos kafka-zookeeper --name=<service name> update start --options=kerberos-toggle-step-3.json
+$ dcos kafka-zookeeper --name=<service name> plan show deploy
+deploy (serial strategy) (COMPLETE)
+└─ node-update (serial strategy) (COMPLETE)
+   ├─ zookeeper-0:[server, metrics] (COMPLETE)
+   ├─ zookeeper-1:[server, metrics] (COMPLETE)
+   └─ zookeeper-2:[server, metrics] (COMPLETE)
+```
+{{ model.techName }} will now require Kerberos authentication for the entire leader election process.
+
+The final step is to require Kerberos authentication for clients connecting to the DC/OS {{ model.techName }} instance with an options file (say `kerberos-toggle-step-4.json`) as follows:
+```json
+{
+    "service": {
+        "security": {
+            "kerberos": {
+                "advanced": {
+                    "required_for_quorum_learner": true,
+                    "required_for_quorum_server": true,
+                    "required_for_client": true
+                }
+            }
+        }
+    }
+}
+```
+which is deployed:
+```bash
+$ dcos kafka-zookeeper --name=<service name> update start --options=kerberos-toggle-step-3.json
+$ dcos kafka-zookeeper --name=<service name> plan show deploy
+deploy (serial strategy) (COMPLETE)
+└─ node-update (serial strategy) (COMPLETE)
+   ├─ zookeeper-0:[server, metrics] (COMPLETE)
+   ├─ zookeeper-1:[server, metrics] (COMPLETE)
+   └─ zookeeper-2:[server, metrics] (COMPLETE)
+```
+
+Unauthenticated clients will now only be allowed to ping, create a session, close a session, or authenticate when communicating with the {{ model.techName }} instance.
```

Comment:
Rewrite as
`Now, only unauthenticated clients will be allowed to ping, create a session, close a session, or authenticate when communicating with the {{ model.techName }} instance.`